docs/guide/faq.rst

```diff
@@ -16,20 +16,20 @@ For example, our example HTTP service from :doc:`the tutorial
 <../tutorial/index>` has an entrypoint function (``make_wsgi_app()``) that sets
 up the application:
 
-.. literalinclude:: ../tutorial/chapter4/sql.py
+.. literalinclude:: ../tutorial/chapter4/helloworld.py
```

Comment:
note: i renamed all the files the same thing (within their separate folders) so that it's easy to `baseplate-serve` in each folder to test the tutorial out